cmd/crio-status/main.go

```diff
@@ -16,13 +16,15 @@ const (
 	defaultSocket = "/var/run/crio/crio.sock"
 	idArg         = "id"
 	socketArg     = "socket"
+
+	deprecatednotice = " [Deprecation Warning: This command has been deprecated and will be removed in future versions. Please use `crio status` instead.]"
```

Comment:
nit: I think we could use a newline here so it reads like:
```
A tool for CRI-O status retrieval
[Deprecation Warning: This command has been deprecated and will be removed in future versions. Please use `crio status` instead.]
```

Reply:
Done.